pages/dkp/kommander/2.2/insights/insights-view/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: Viewing insight items
+title: Viewing Insight Items
+excerpt: Viewing and Troubleshooting Insight Items
+menuWeight: 30
+beta: false
+enterprise: false
+techPreview: true
+---
+
+<p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis.</p>
+
+After the [Insights Engine is enabled](../insights-setup/), select one of your workspaces to view your insight items. From a Workspace Dashboard, the Insights summary table displays the most recent insight items, below the workspace summary cards. You can toggle the view between Critical, Warning, Notices, and View All for a summary of insight items.
```

Comment:
@gvasquezvargas - this is what we were talking about earlier - "From a Workspace Dashboard..."